autofix/rules/empty-line-between-blocks.js

```diff
@@ -7,6 +7,18 @@ function emptyLineBetweenBlocks(tree) {
     if (node.is('ruleset') && index > 0) {
       const prev = parent.get(index - 1);
 
+      // fix only if any of leading elements is anything than space
```

Comment:
@jedrula When it comes to comment, it should mean that we shouldn't proceed if any of previous siblings of current node is not `space`(which also contains new-lines) (for example `ruleset`, `definition`).

I cannot think of anything simpler than this while 😿